airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -125,15 +129,16 @@ def get_dag_runs(
 
 def _fetch_dag_runs(
     query,
-    end_date_gte,
-    end_date_lte,
-    execution_date_gte,
-    execution_date_lte,
-    start_date_gte,
-    start_date_lte,
-    limit,
-    offset,
-    order_by,
+    end_date_gte: Optional[str],
+    end_date_lte: Optional[str],
+    execution_date_gte: Optional[str],
+    execution_date_lte: Optional[str],
+    start_date_gte: Optional[str],
+    start_date_lte: Optional[str],
+    state: Optional[List[str]],
+    limit: Optional[int],
+    offset: Optional[int],
+    order_by: str,
```

Comment:
I added type annotations to the functions I affected, but did not touch the others.